app/views/get-started/deploy-your-prototype/README.md

```diff
@@ -0,0 +1,87 @@
+This guide explains how to deploy prototypes to MoJ's Heroku account.
+
+After following this guide your prototype will automatically deploy any changes that you push to GitHub.
+
+## Before you start
+
+You'll need to be a member of the [moj-design](https://dashboard.heroku.com/teams/moj-design/apps) Heroku team and have your prototype in [MoJ's GitHub account](https://github.com/ministryofjustice).
+
+If you don't have access to these, ask in the Slack channel [#prototyping](https://mojdt.slack.com/messages/C9DSD7NA0) and whoever is managing support that day should be able to add you.
+
+If you don't know how to setup GitHub read the [version your prototype](#) guide.
+
+## Setting up Heroku
+
+We use pipelines in Heroku. Pipelines let you have multiple apps and keep things together in one place.
```

Comment:
Should we be encouraging pipelines or more simply, apps? I understand the value of the former, but this may be too technical for designers, or unnecessary.